### PR TITLE
Update Bio

### PR DESCRIPTION
**Pull request by s4shantanu**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3769"></a>

### CODE REVIEW

File: data/s4shantanu.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/s4shantanu/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/shantanu-raut/",
+      "icon": "FaLinkedin"
+    }
```
Comment (Abdulwahid7027) [inline data/s4shantanu.json:22]: Add comma after curly braces and looks like this ` } ,`
```suggestion
    } ,
```

**Review by Abdulwahid7027 — CHANGES_REQUESTED** [review]

File: data/s4shantanu.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "name": "Shantanu Raut",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "I am Shantanu Raut, First year MCA student from GH Raisoni University. I'm a specialist in Backend development using Python technology. I did an internship at iris business services Pvt Ltd. I am proficient in Python, Java, and JavaScript technology.",
-  "avatar": "https://github.com/s4shantanu.png",
+  "bio": "Graduate || Backend Developer || Python || Java | |DSA || Leetcode 2🌟 || Hackerrank 5🌟 || Open-source || Postman Student Expert || Technical Volunteer in All About Programming || GDSC Member",
+  "avatar": "https://github.com/s4shantanu.png"
```
Comment (amandamartin-dev) [inline data/s4shantanu.json:6]: missing a comma here

File: data/s4shantanu.json [hunk 3]
```diff
@@ -19,8 +19,10 @@
       "name": "Follow me on Linkedin",
       "url": "https://www.linkedin.com/in/shantanu-raut/",
       "icon": "FaLinkedin"
-    },
+
```
Comment (amandamartin-dev) [inline data/s4shantanu.json:22]: this extra space can be removed

File: data/s4shantanu.json [hunk 4]
```diff
@@ -19,8 +19,10 @@
...
+    } ,
     {
+
```
Comment (amandamartin-dev) [inline data/s4shantanu.json:25]: this extra space can be removed

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

**Comment by s4shantanu** [comment]
Hi,  Please Accept my PR!!

**Review by amandamartin-dev — APPROVED** [review]
LGTM! Thanks for contributing!

**Comment by s4shantanu** [comment]
Thank You @amandamartin-dev

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**